doc/classes/ProjectSettings.xml

```diff
@@ -1026,6 +1026,10 @@
 		</member>
 		<member name="editor/version_control/plugin_name" type="String" setter="" getter="" default="&quot;&quot;">
 		</member>
+		<member name="filesystem/gdscript/inline_script_uids" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], the UIDs of [GDScript] resources will be stored inside the scripts themselves, inside a comment on their first line. UID comments will be automatically created when the scripts are saved. If [code]false[/code], the UIDs are stored in dedicated files. See also [ResourceUID].
+			Note that if some scripts already have associated UID files, you will need to manually delete them after enabling this setting. The files take priority in determining the script's UID.
```

Comment:
```suggestion
			[b]Note:[/b] After enabling this setting, if some scripts already have associated UID files they may need to be manually deleted. The files take priority in determining the script's UID.
```
Also, I get why myself, but this behavior can be seen as perplexing especially given how alienating UIDs are to non-advanced users.

Reply:
Well, the setting *is* for advanced users.